### DISCUSSION TITLE
potential funding opportunities for projects using SPECFEM

### DISCUSSION THREAD

**Discussion by carltape** [SPECFEM/specfem3d#1569] [General]
Here are some NSF programs where SPECFEM could fit in:
CSSI: https://www.nsf.gov/pubs/2022/nsf22632/nsf22632.htm
POSE: https://www.nsf.gov/pubs/2022/nsf22572/nsf22572.htm
SCIPE: https://www.nsf.gov/pubs/2023/nsf23521/nsf23521.htm
CyberTraining: https://www.nsf.gov/pubs/2023/nsf23520/nsf23520.htm
These were shared today at the CIG Annual Business Meeting. It would be interesting to tabulate all the funding programs and paths that led to the SPECFEM publications. I suspect that most development is achieved through service, whether formally part of one's job or through "free" time. The topic of how to sustain funding for efforts like SPECFEM is a massive one that is also considered by [Seismological Software Initiative](https://jabrackenhoff.gitlab.io/seismological-software-initiative/ ).

Please chime in here if you see promising funding opportunities.

**Comment by ljhwang** [comment]
Also from our website, navigate to Resources > Funding Opportunities which links to the following NSF website:

	https://www.nsf.gov/geo/geo-ci/index.jsp
